### PR TITLE
[#3] Implement UI for Login screen

### PR DESCRIPTION
**Pull request by chornerman**
#3 

## What happened 👀

Implement UI for the Login screen

## Insight 📝

- [x] Add resource files to use in the Login screen (font, background, header icon)
- [x] Implement UI for Login screen
- [x] Set up localization using `intl `
- [x] Remove unit test for now

### CODE REVIEW

File: lib/pages/login/widgets/text_input_widget.dart [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+                onPressed: () {
+                  // TODO: Define in integration part
+                },
+                style: TextButton.styleFrom(
+                  minimumSize: Size.zero,
+                  padding: EdgeInsets.zero,
+                  tapTargetSize: MaterialTapTargetSize.shrinkWrap,
+                ),
+                child: Text(
+                  "Forgot?",
```
Comment (markgravity) [inline lib/pages/login/widgets/text_input_widget.dart:58]: Can you support localization for this text? 🙏

**Review by markgravity — COMMENTED** [review]
Please support `x2`, `x3` version for `.png` images 🙏 
More info [here](https://docs.flutter.dev/development/ui/assets-and-images#loading-images)

File: lib/pages/login/widgets/text_input_widget.dart (hunk 1 above)
Comment (chornerman) [reply lib/pages/login/widgets/text_input_widget.dart]: @markgravity Updated 🙏🏼

**Review by markgravity — APPROVED** [review]
LGTM now 👍

File: lib/app_colors.dart [hunk 2]
```diff
@@ -0,0 +1,5 @@
+import 'package:flutter/material.dart';
+
+class AppColors {
+  static const Color chineseBlack = Color(0xff15151A);
```
Comment (luongvo) [inline lib/app_colors.dart:4]: Optional: I think you can generate color from xml assets as well by https://pub.dev/packages/flutter_gen 🔥

File: lib/assets.dart [hunk 3]
```diff
@@ -0,0 +1,4 @@
+class Assets {
+  static const String loginBackground = "assets/images/bg_login.png";
+  static const String nimbleIcon = "assets/images/ic_nimble.svg";
```
Comment (luongvo) [inline lib/assets.dart:3]: same, apply https://pub.dev/packages/flutter_gen

File: lib/fonts.dart [hunk 4]
```diff
@@ -0,0 +1,3 @@
+class Fonts {
+  static String neuzeit = "Neuzeit";
```
Comment (luongvo) [inline lib/fonts.dart:2]: apply https://pub.dev/packages/flutter_gen

File: lib/pages/login/widgets/text_input_widget.dart [hunk 5]
```diff
@@ -0,0 +1,71 @@
...
+              style: const TextStyle(
+                fontSize: 17,
+                color: Colors.white,
+              ),
+              decoration: InputDecoration(
+                hintText: hintText,
+                hintStyle: TextStyle(
+                  fontSize: 17,
+                  color: Colors.white.withOpacity(0.2),
+                ),
```
Comment (luongvo) [inline lib/pages/login/widgets/text_input_widget.dart:41]: I think we should try to define TextStyle through `ThemeData`

File: lib/pages/login/widgets/text_input_widget.dart [hunk 6]
```diff
@@ -0,0 +1,71 @@
...
+                child: Text(
+                  AppLocalizations.of(context)!.loginForgotPassword,
+                  style: TextStyle(
+                    fontSize: 15,
+                    color: Colors.white.withOpacity(0.3),
+                  ),
+                ),
+              ),
+            ),
```
Comment (luongvo) [inline lib/pages/login/widgets/text_input_widget.dart:66]: Should we move this widget out of the custom input? I feel like we're putting specific logic in a common thing. Should we define an `endWidget` instead? 🤔 

```
class TextInputWidget extends StatelessWidget {
  final String hintText;
  final bool isPasswordInput;
  final Widget? endWidget;

  const TextInputWidget({
    Key? key,
    required this.hintText,
    this.isPasswordInput = false,
    this.endWidget = null,
  }) : super(key: key);


  @override
  Widget build(BuildContext context) {
    return Container(
      height: Dimens.space56,
      padding: const EdgeInsets.symmetric(horizontal: Dimens.space12),
      decoration: BoxDecoration(
        color: Colors.white.withOpacity(0.1),
        borderRadius: BorderRadius.circular(Dimens.space12),
      ),
      child: Row(
        children: [
            ...
            if (endWidget != null) endW
```

File: pubspec.yaml [hunk 7]
```diff
@@ -29,11 +29,11 @@ environment:
...
   flutter:
     sdk: flutter
-
-
-  # The following adds the Cupertino Icons font to your application.
-  # Use with the CupertinoIcons class for iOS style icons.
+  flutter_localizations:
+    sdk: flutter
+  intl: ^0.17.0
   cupertino_icons: ^1.0.2
```
Comment (luongvo) [inline pubspec.yaml:35]: is this redundant now?

**Review by luongvo — COMMENTED** [review]
some my first reviews @chornerman, good work 👏

**Pull request merged**